src/applications/check-in/locales/en/translation.json

```diff
@@ -290,8 +290,11 @@
   "file-a-travel-reimbursement-claim-if-youre-eligible": "File a travel reimbursement claim if you’re eligible for health care travel reimbursement. First, we need your last name and date of birth to make sure it’s you.",
   "were-processing-your-travel-claim_one": "We’re processing your travel reimbursement claim",
   "were-processing-your-travel-claim_other": "We’re processing your travel reimbursement claims",
-  "this-claim-is-for-your-appointment_one": "This claim is for your appointment on {{date, long}} at",
-  "this-claim-is-for-your-appointment_other": "These claims are for appointments on {{date, long}} at",
+  "this-claim-is-for-your-appointment": "$t(claims, {\"count\": {{claims}} }) for your $t(appointments, {\"count\": {{appointments}} }) on {{date, long}} at",
```

Comment:
I had no idea you could do translations within translations. Two questions about this:

- Does the order of the internal translation matter (should it be BEFORE the item that uses it?
- Is it a good idea? I know it shortens having to do text, but without better ways of knowing exactly what is tied to what (like an indented group or something) I feel like it introduces future maintenance issues when updating (possibly)

Reply:
- The order doesn't matter since the JSON file is already parsed ahead of time. So it's not like defining vars in code.
- I think it is okay from a maintainability POV. Holding to our rules on changing translations, anything new would need a new key and new value. I think we could keep these related keys together when alphabetizing. But also I think you bring up a good point around relationship. I'm going to change the keys to feel like children of this key.